packages/v4/src/content/design-guidelines/styles/typography/typography.md

```diff
@@ -41,29 +41,33 @@ Use our typographic styles to communicate visual hierarchy. A consistent and log
   styleProps = {styleProps.tiny} />
 
 ## Line height
-<div style={{marginBottom: '32px'}}>When laying out text-based content, you need to factor in line-height when measuring spacing. Line height is a relative number used in CSS that represents a ratio to the text size. For example, body text uses a line height of 1.5. Body text size is 16px. 16*1.5 = 24px line height. So, when creating designs in design software, you must include the 24px line height as a part of the text.</div>
+<div style={{marginBottom: '32px'}}>Accommodating for line-height is important when measuring spacing in text-based content. 
```

Comment:
We have a hyphen for "line-height" here, but not anywhere else. For consistency, let's go with "line height" 

"Accommodating for line height" sounds a little confusing. I recommend restructuring this so that we give the user an actionable step: "When measuring spacing in text-based content, plan for line height."